poznaj/templates/main.html

```diff
@@ -0,0 +1,135 @@
+{% load static i18n %}
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>PoznajApp | Poznaj swoje miasto</title>
+
+    <!-- Site CSS -->
+    <link rel="stylesheet" href="{% static 'vendor/bootstrap/css/bootstrap.min.css' %}">
```

Comment:
Can we serve these via some kind of CDN?

Reply:
Can. But what is CDN? :D

Reply:
CDN: https://pl.wikipedia.org/wiki/Content_Delivery_Network. Instead us they host js & css